pyrelacs/DataClasses/RelacsFile.py

```diff
@@ -138,11 +137,11 @@ def parse_structure(filename, verbose=False):
                     structure.append(FileRange(start, line_no, 'data'))
                     within_data_block = False
                     if verbose: print("DATA END", line[:20], line_no)
-                elif within_meta_block:
+                if within_meta_block:
```

Comment:
argh...

man, another of those. I reverted few changes, but this one is one that 
I did not.
Keep the original, the whole def parse_structure as it is.

Sorry Fabian.

On 11.03.16 15:59, Fabian Sinz wrote:
>
> In pyrelacs/DataClasses/RelacsFile.py 
> <https://github.com/relacs/pyrelacs/pull/16#discussion_r55839297>:
>
> > @@ -138,11 +137,11 @@ def parse_structure(filename, verbose=False):
> >                      structure.append(FileRange(start, line_no, 'data'))
> >                      within_data_block = False
> >                      if verbose: print("DATA END", line[:20], line_no)
> > -                elif within_meta_block:
> > +                if within_meta_block:
>
> Trying understand the logic. Why replace it with |if| here?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/relacs/pyrelacs/pull/16/files#r55839297>.
>